modules/programs/fish.nix

```
@@ -306,17 +306,21 @@ in {
 
         generateCompletions = package:
           pkgs.runCommand "${package.name}-fish-completions" {
-            src = package;
+            srcs = [ package ] ++ lib.filter (p: p != null)
```

The `lib.` prefix shouldn't be necessary
```suggestion
            srcs = [ package ] ++ filter (p: p != null)
```